packages/Algernon.package/ALGActionVerbFilter.class/class/keywords.st

```diff
@@ -2,4 +2,6 @@ filtering
 keywords
 	^ Set newFrom: {'run'. 'browse'. 'fileout'. 
 		'open'. 'saverepo'. 'do'. 
-		'inspect'. 'explore'. '=' }
\ No newline at end of file
+		'inspect'. 'explore'. '='.
+		'front'. 'expand'. 'collapse'. 
```

Comment:
I think it would be nice if we add some helptexts for those new keywords in the `helpTexts` method that is also located in the ActionVerbFilter class. Adding those explainations to the [Keywords Wiki Page](https://github.com/hpi-swa-teaching/Algernon-Launcher/wiki/Keywords) would be the next step then :)

Reply:
Ah, you're right, we totally forgot about these🙈
I can add these to the wiki page as soon as this PR is merged if that's okay?